internal/cli/atlas/events/orgs_list.go

```diff
@@ -44,11 +46,23 @@ func (opts *orgListOpts) initStore(ctx context.Context) func() error {
 }
 
 func (opts *orgListOpts) Run() error {
-	listOpts := opts.newEventListOptions()
-
-	var r *atlas.EventResponse
+	var r interface{}
 	var err error
-	r, err = opts.store.OrganizationEvents(opts.ConfigOrgID(), listOpts)
+	minDate, _ := time.Parse(time.RFC3339, opts.MinDate)
+	maxDate, _ := time.Parse(time.RFC3339, opts.MaxDate)
+
+	// TODO - event type is array but we expect single event
+	eventType, _ := mongodbatlasv2.NewEventTypeForOrgFromValue(opts.EventType[0])
+	listEventsAPIParams := mongodbatlasv2.ListOrganizationEventsApiParams{
+		OrgId:        opts.OrgID,
```

Comment:
wrong orgID this won't leverage env variables or config file

Reply:
Good catch.